.github/workflows/ci.yml

```diff
@@ -17,8 +17,9 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.1.0
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.2.0
     with:
       fastify-dependency-integration: true
       license-check: true
       lint: true
+      node-versions: '[16, 18, 20]'
```

Comment:
Why?

Reply:
I’ll remove it, by default I was adding it to libraries repos